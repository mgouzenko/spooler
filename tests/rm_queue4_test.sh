addqueue tests/test_dir/cat
addqueue tests/test_dir/world
rmqueue 2 1
