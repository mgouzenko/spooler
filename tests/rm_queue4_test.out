tests/test_dir/cat: Y 1
tests/test_dir/world: Y 2
2: Y
1: Y
